File: 1584-average-salary-excluding-the-minimum-and-maximum-salary/1584-average-salary-excluding-the-minimum-and-maximum-salary.cs
public class Solution {
    public double Average(int[] salary) {
        double sum = 0;

        for(int i = 0; i < salary.Length; i++){
            sum += salary[i];
        }
        sum -= salary.Max();
        sum -= salary.Min();
        
        return sum / (salary.Length - 2);
    }
}

File: 1584-average-salary-excluding-the-minimum-and-maximum-salary/1584-average-salary-excluding-the-minimum-and-maximum-salary.js
/**
 * @param {number[]} salary
 * @return {number}
 */
var average = function(salary) {
    var sum = 0;
    for(let i = 0; i < salary.length; i++){
        sum += salary[i];
    }
    sum -= Math.max(...salary)
    sum -= Math.min(...salary)
    return sum / (salary.length - 2)
};